src/Benchmarks/BenchParams.cpp

```diff
@@ -151,7 +151,9 @@ inline shared_ptr<Params> decideBenchParams(GenSync::SyncProtocol syncProtocol,
         is >> *par;
         return par;
     } else if (syncProtocol == GenSync::SyncProtocol::FullSync) {
-        return nullptr;
+        auto par = make_shared<FullSyncParams>();
```

Comment:
Has this been tested?

Reply:
There is no specific (unit) test for this, or anything regarding the decideBenchParams function as far as I can see. Only Runner.cpp (the Benchmarks executable) reads benchmark parameters from a file. I have tested FullSync using the Benchmarks executable, which is what this enables.
At any rate, returning nullptr here leads to a crash in Benchmarks when trying to use FullSync because it never checks if BenchParams.syncParams is nullptr.